### ISSUE TITLE
Error running ft_qualitycheck

### ISSUE THREAD

**Issue by suxpert** [fieldtrip/fieldtrip#623]
Running the following code will lead to an error
``` matlab
cfg = [];
cfg.dataset = 'task1.vhdr';
[info, timelock, freq, summary] = ft_qualitycheck(cfg);
```
After several minutes, matlab complained
> Undefined function or variable 'headpos'

Then I looked into the source code, and found that this error is due to
> Reference to non-exist field 'dataset'

at line 421 of [ft_qualitycheck](https://github.com/fieldtrip/fieldtrip/blob/master/ft_qualitycheck.m#L421)

I manually add this field to `timelock.cfg`, the code can run without error, but with a strange report:

![task1](https://user-images.githubusercontent.com/1191414/34345365-53767e7e-ea29-11e7-80c4-5fc9ed643e39.png)

Did I mess up something?

**Comment by suxpert** [comment]
BTW, I was using the latest version from github (2017.12.26 UTC)
> $ git pull
Already up-to-date.

**Comment by StolkArjen** [comment]
Thanks for the bug report. The 'strange' output is a summary of the data and is intended to look like this. For MEG data, it would show more information such as the head position dynamics.

The bug you ran into is not intended. I think it is fixed now, as soon as the above PR (626) gets pulled in. In essence, I changed line 412 into:
[iseeg, ismeg, isctf, fltp] = filetyper(info.hdr.orig.FileName);

which worked for the few different datasets I tested.
